openbas-api/src/main/java/io/openbas/rest/atomic_testing/form/InjectStatusOutput.java

```diff
@@ -4,18 +4,18 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 import io.openbas.database.model.ExecutionStatus;
-import io.openbas.database.model.ExecutionTraces;
 import jakarta.validation.constraints.NotNull;
 import java.time.Instant;
 import java.util.ArrayList;
 import java.util.List;
 import lombok.Builder;
 import lombok.Getter;
 import lombok.Setter;
+import lombok.experimental.SuperBuilder;
 
 @Setter
 @Getter
-@Builder
+@SuperBuilder
```

Comment:
Can you take this opportunity to remove @ getter and @ setter and replace them by @ data ?

Reply:
Also, is SuperBuilder necessary instead of Builder ? AFAIK, is t's only usefull when classes extends another.